app/models/grda_warehouse/hud/project_coc.rb

```diff
@@ -55,6 +55,12 @@ class ProjectCoc < Base
       where(arel_table[:City].lower.in(place.map(&:downcase)))
     end
 
+    scope :in_county, ->(county:) do
+      county = Array(county)
+      # FIXME
```

Comment:
What needs fixing?

Reply:
So, we need to actually figure out how to calculate county (which is why the next comment you had is disabled.)  Holding off on merging this until we have that settled.